app/main.py

```diff
@@ -17,13 +17,12 @@ def die(self) -> None:
             Animal.alive.remove(self)
 
     def __repr__(self) -> str:
-        return (
-                "{" +
-                "Name: " + str(self.name) + ", " +
-                "Health: " + str(self.health) + ", " +
-                "Hidden: " + str(self.hidden) +
-                "}"
-        )
+        return ("{"
+                + "Name: " + str(self.name) + ", "
+                + "Health: " + str(self.health) + ", "
+                + "Hidden: " + str(self.hidden)
+                + "}")
```

Comment:
I guess comment was that you don't need f-strinf in every lane, so you can use f-strings where needed, but pass regular strings in other cases.